webpack/make-webpack-config.js

```diff
@@ -0,0 +1,191 @@
+var packageInfo = require('../package');
+var path = require('path');
+/*
+  Plugins
+ */
+var autoprefixer = require('autoprefixer');
+var colorLighten = require('less-color-lighten');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var StringReplacePlugin = require('string-replace-webpack-plugin');
+
+var analyticsKey = '39uhSEOoRHMw6cMR6st9tYXDbAL3JSaP';
+
+var VARS = {
+  VERSION: packageInfo.version,
+  ENV: process.env.NODE_ENV,
+  ANALYTICS_KEY: analyticsKey
+}
+
+function absPath() {
+  var args = [].slice.apply(arguments);
+  args.unshift(__dirname, '..');
+
+  return path.resolve.apply(path.resolve, args);
+}
+
+function extractForProduction(loaders) {
+  return ExtractTextPlugin.extract('style', loaders.substr(loaders.indexOf('!')));
+}
+
+module.exports = function (options) {
+  var cssLoaders = 'style!css!postcss';
+  var scssLoaders = cssLoaders + '!sass';
+  var sassLoaders = scssLoaders + '?indentedSyntax=sass';
+  var lessLoaders = cssLoaders + '!less';
+
+  if (options.production) {
+    cssLoaders = extractForProduction(cssLoaders);
+    sassLoaders = extractForProduction(sassLoaders);
+    scssLoaders = extractForProduction(scssLoaders);
+    lessLoaders = extractForProduction(lessLoaders);
+
+    analyticsKey = '51ybGTeFEFU1xo6u10XMDrr6kATFyRyh';
```

Comment:
Moved them to package.json. Unfortunately they need to be kept somewhere with global context because they are used in multiple places. Dependents should be consolidated but lets worry about that later. package.json seems like a good candidate for this configuration for now.